arrow/parser.py

```diff
@@ -218,7 +308,9 @@ def parse(self, datetime_string, fmt, normalize_whitespace=False):
             return self._parse_multiformat(datetime_string, fmt)
 
         try:
-            fmt_tokens, fmt_pattern_re = self._generate_pattern_re(fmt)
+            fmt_tokens, fmt_pattern_re = self._generate_pattern_re(
+                fmt
+            )  # type: List[_FORMAT_TYPE], Pattern[str]
```

Comment:
This can be changed to a type annotation right before the unpacking:

```python
fmt_tokens: List[_FORMAT_TYPE]
fmt_pattern_re: Pattern[str]
fmt_tokens, fmt_pattern_re = self._generate_pattern_re(
    fmt
)
```

Reply:
Done!